src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/OtherAccountingPolicyServiceImpl.java

```diff
@@ -0,0 +1,91 @@
+
+    @Autowired
+    private AccountingPoliciesService accountingPoliciesService;
+
+    @Autowired
+    private AccountingPoliciesTransformer accountingPoliciesTransformer;
+
+    private static final String OTHER_ACCOUNTING_POLICY_FIELD_PATH =
+        "otherAccountingPolicyDetails";
+
+    private static final String INVALID_STRING_SIZE_ERROR_MESSAGE =
+        "validation.length.minInvalid.accounting_policies.other_accounting_policy";
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public OtherAccountingPolicy getOtherAccountingPolicy(String transactionId,
+        String companyAccountsId) throws ServiceException {
+        AccountingPoliciesApi accountingPoliciesApi =
+            accountingPoliciesService.getAccountingPoliciesApi(transactionId, companyAccountsId);
+        return accountingPoliciesTransformer.getOtherAccountingPolicy(accountingPoliciesApi);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public List<ValidationError> submitOtherAccountingPolicy(String transactionId,
+        String companyAccountsId, OtherAccountingPolicy otherAccountingPolicy)
+        throws ServiceException {
+        List<ValidationError> validationErrors = validateOtherAccountingPolicy(
+            otherAccountingPolicy);
+        if (!validationErrors.isEmpty()) {
+            return validationErrors;
+        }
+        AccountingPoliciesApi accountingPoliciesApi =
+            accountingPoliciesService.getAccountingPoliciesApi(transactionId, companyAccountsId);
+        accountingPoliciesTransformer
+            .setOtherAccountingPolicy(otherAccountingPolicy, accountingPoliciesApi);
+        return accountingPoliciesService
+            .updateAccountingPoliciesApi(transactionId, companyAccountsId, accountingPoliciesApi);
+    }
+
+    /**
+     * Validate a submitted other accounting policy
+     *
+     * @param otherAccountingPolicy The policy to validate
+     * @return a list of validation errors, or an empty array list if none are present
+     */
+    private List<ValidationError> validateOtherAccountingPolicy(
+        OtherAccountingPolicy otherAccountingPolicy) {
+        List<ValidationError> validationErrors = new ArrayList<>();
+        if (otherAccountingPolicyNotProvided(otherAccountingPolicy)) {
+            ValidationError validationError = new ValidationError();
+            validationError.setFieldPath(OTHER_ACCOUNTING_POLICY_FIELD_PATH);
+            validationError.setMessageKey(INVALID_STRING_SIZE_ERROR_MESSAGE);
+            validationErrors.add(validationError);
+        }
+        return validationErrors;
+    }
+
+    /**
+     * Check whether a other accounting policy has not been provided
+     *
+     * @param otherAccountingPolicy The policy to check
+     * @return true if the 'include policy' boolean is set to true, but no details are provided
+     */
+    private boolean otherAccountingPolicyNotProvided(
```

Comment:
This could possibly be named `isOtherAccountingPolicyNotProvided` - javadoc may also be unnecessary as the method content and name is self explanatory.